lib/node_modules/@stdlib/namespace/alias2pkg/data/data.csv

```diff
@@ -833,8 +946,27 @@ base.random.t,"@stdlib/random/base/t"
 base.random.triangular,"@stdlib/random/base/triangular"
 base.random.uniform,"@stdlib/random/base/uniform"
 base.random.weibull,"@stdlib/random/base/weibull"
+base.rcbrt,"@stdlib/math/base/special/rcbrt"
+base.rcbrtf,"@stdlib/math/base/special/rcbrtf"
 base.reldiff,"@stdlib/math/base/utils/relative-difference"
+base.removeFirst,"@stdlib/string/base/remove-first"
+base.removeFirstCodePoint,"@stdlib/string/base/remove-first-code-point"
+base.removeFirstGraphemeCluster,"@stdlib/string/base/remove-first-grapheme-cluster"
+base.removeLast,"@stdlib/string/base/remove-last"
+base.removeLastCodePoint,"@stdlib/string/base/remove-last-code-point"
+base.removeLastGraphemeCluster,"@stdlib/string/base/remove-last-grapheme-cluster"
 base.rempio2,"@stdlib/math/base/special/rempio2"
+base.repeat,"@stdlib/string/base/repeat"
+base.replace,"@stdlib/string/base/replace"
```

Comment:
@Planeshifter Some of these are not correct. For example, we should mirror top-level aliases for "base" aliases: https://github.com/stdlib-js/stdlib/blob/b5c21390fd63572bf348a178a2f816d5702bf785/lib/node_modules/%40stdlib/namespace/lib/namespace/r.js#L688.